### PR TITLE
Easy

### PR DESCRIPTION
**Pull request by andrisLaduzans**

### CODE REVIEW

File: exercises/easy/Acronym/index.js [hunk 1]
```diff
@@ -6,6 +6,17 @@
...
-function parse(input) {}
-
+function parse(input) {
+  function cleanString(string) {
+    const alphaOnly = string.replace(/[^a-zA-Z']/gi, " ");
+    return alphaOnly.toLowerCase();
+  }
+  let inpArr = cleanString(input);
+  inpArr = inpArr.split(" ");
+  let Acronyms = "";
```
Comment (sandris-) [inline exercises/easy/Acronym/index.js]: uppercase?

File: exercises/easy/Anagrams/index.js [hunk 2]
```diff
@@ -12,6 +12,19 @@
  * anagrams('Hi there', 'Bye there') === false
  */
 
-function anagrams(stringA, stringB) {}
-
+function anagrams(stringA, stringB) {
+  //sakumaa notiram virknes
```
Comment (sandris-) [inline exercises/easy/Anagrams/index.js]: what is this language?

File: exercises/easy/Anagrams/index.js [hunk 3]
```diff
@@ -12,6 +12,19 @@
...
+    const alphaOnly = string.replace(/[^a-zA-Z]/gi, "");
+    return alphaOnly.toLowerCase();
+  }
+  stringA = cleanString(stringA);
+  stringB = cleanString(stringB);
+  stringA = Array.from(stringA).sort();
+  stringB = Array.from(stringB).sort();
+  stringA = JSON.stringify(stringA);
+  stringB = JSON.stringify(stringB);
+  const EqOrNot = stringA == stringB;
```
Comment (sandris-) [inline exercises/easy/Anagrams/index.js]: upper?

File: exercises/easy/HighScores/index.js [hunk 4]
```diff
@@ -9,11 +9,32 @@ class HighScores {
...
+      if (currentScore > best) {
+        best = currentScore;
+      }
+    }
+    return best;
+  }
 
-  get personalTopThree() {}
+  get personalTopThree() {
+    //var saglabāt masīvu, sasortēt, un pēc tam izvadīt pirmos trīs.
```
Comment (sandris-) [inline exercises/easy/HighScores/index.js]: ?

File: exercises/easy/IsEven/index.js [hunk 5]
```diff
@@ -10,6 +10,12 @@
  * isEven(3) === false
  */
 
-function isEven(n) {}
+function isEven(n) {
+  // idejas skaidrojums :
+  // https://www.includehelp.com/c-programs/check-even-odd-without-arithmetic-relational-operator.aspx
+  // mēs par šo vēl neesam mācījušies kursos,
```
Comment (sandris-) [inline exercises/easy/IsEven/index.js]: ?

File: exercises/easy/TwoFer/index.js [hunk 6]
```diff
@@ -12,6 +12,14 @@
  * twoFer() === "One for you, one for me."
  */
 
-function twoFer(name) {}
+// function twoFer(name) {
```
Comment (sandris-) [inline exercises/easy/TwoFer/index.js]: ?

File: exercises/easy/Acronym/index.js (hunk 1 above)
Comment (andrisLaduzans) [reply exercises/easy/Acronym/index.js]: line 18 i take every word's charAt[0] toUpperCase and save that char to string "Acronyms"

File: exercises/easy/Anagrams/index.js (hunk 2 above)
Comment (andrisLaduzans) [reply exercises/easy/Anagrams/index.js]: sory

File: exercises/easy/Anagrams/index.js (hunk 3 above)
Comment (andrisLaduzans) [reply exercises/easy/Anagrams/index.js]: why upper?, i save all strings as lowecase, sort both, and then check whether they are identical.

Comment (andrisLaduzans) [reply exercises/easy/Anagrams/index.js]: no need for upperCase , i clean all strings from any symbols/numbers
save them to lowercase, after that i sort them a->z, join them back to single word ,
then compare if the result is true both words had identical characters

File: exercises/easy/HighScores/index.js (hunk 4 above)
Comment (andrisLaduzans) [reply exercises/easy/HighScores/index.js]: sory, forgot some comments

File: exercises/easy/IsEven/index.js (hunk 5 above)
Comment (andrisLaduzans) [reply exercises/easy/IsEven/index.js]: thoght i should reference where my solution came from..

File: exercises/easy/TwoFer/index.js (hunk 6 above)
Comment (andrisLaduzans) [reply exercises/easy/TwoFer/index.js]: removed comment

Comment (andrisLaduzans) [reply exercises/easy/TwoFer/index.js]: deleted the comment

File: exercises/easy/Anagrams/index.js (hunk 3 above)
Comment (sandris-) [reply exercises/easy/Anagrams/index.js]: why variable name is uppercase?

File: exercises/easy/IsEven/index.js (hunk 5 above)
Comment (sandris-) [reply exercises/easy/IsEven/index.js]: use english if you want to reference smth

**Pull request merged**